lib/plugins/install/tests/install.js

```diff
@@ -77,5 +129,18 @@ describe('Install', () => {
         expect(downloadStub.args[0][0]).to.equal(`${install.options.url}/archive/master.zip`);
       });
     });
+
+    it('should download and rename the service based on the GitHub URL', () => {
```

Comment:
Can we enhance this test somehow that the call to the renaming method is tested as well?